src/api.rs

```diff
@@ -71,7 +73,7 @@ impl<'a, 'r> FromRequest<'a, 'r> for Auth {
 	type Error = ();
 
 	fn from_request(request: &'a Request<'r>) -> request::Outcome<Self, ()> {
-		let mut cookies = request.guard::<Cookies>().unwrap();
+		let mut cookies = request.guard::<Cookies<'_>>().unwrap();
```

Comment:
Are these anonymous lifetimes a new 2018 idiom? I don't like the added noise but I'm happy to keep them if they're an official recommendation.

Reply:
https://doc.rust-lang.org/edition-guide/rust-2018/ownership-and-lifetimes/the-anonymous-lifetime.html